docs/glossary.md

```diff
@@ -0,0 +1,243 @@
+        },
+        // the item SFT you wish to unequip
+        Item: {
+            brand: itemBrand,
+            value: makeCopyBag([[wantedItem, 1n]]),
+        },
+    },
+};
+```
+
+
+#### Marketplace 
+The KREAd contract includes logic for buying and selling Character and Item assets in an unmanaged P2P marketplace. The marketplace can be used via `kread.app/shop` and allows users to list their Characters or Items for sale at a price of their choosing. From there, anyone with enough funds can buy it. Any sale via KREAd marketplace is subject to a platform fee of (3% of sale price) which goes to an address own by KREAd, as well as a royalty fee of (10% of sale price) paid to the artist. 
+
+Keep the following logic in mind when using the KREAd's marketplace:
+- Each market entry can only contain 1 item or character SFT. (In order to sell 10 items, 10 market entries must be listed)
+- Currently, the only supported payment token is IST
+- Selling a character transfers access to its inventory. If you wish to keep the items equipped when selling a character make sure you unequip them beforehand
+- Marketplace listings can be cancelled, so you are free change your mind about selling an asset as long as no one has fulfilled the offer. 
+  
+#### Sell
+KREAd's marketplace supports the sale of both Characters and Items. Calling `makeSellCharacterInvitation` or `makeSellItemInvitation` (a contract public method) returns an invitation to sell, which can be used to form an [Agoric Offer](https://docs.agoric.com/guides/zoe/proposal.html). The sell offer proposal must adhere to the following format:
+```js
+// for makeSellItemInvitation
+const proposal {
+    want : {
+        Price: { brand: IST_BRAND, value: ASKING_PRICE_IN_IST },
+    },
+    give: {
+        Item: { brand: KREAD_ITEM_BRAND, value: ITEM_TO_SELL },
+    }
+}
+
+// for makeSellCharacterInvitation
+const proposal {
+    want : {
+        Price: { brand: IST_BRAND, value: ASKING_PRICE_IN_IST },
+    },
+    give: {
+        Item: { brand: KREAD_CHARACTER_BRAND, value: CHARACTER_TO_SELL },
+    }
+}
+```
+>Assets specified on an Offer's give will be escrowed by Zoe (Agoric's smart contract platform) once they sign the offer transaction, meaning they will no longer be in the user's wallet the unless they cancel the offer before completion.
+
+Once the market listing is created, it will be visible at kread.app/shop, where any user can buy it (provided they have enough IST to cover the set asking price + fees). The user sets the asking price via the offer's want property, then Agoric's [Offer Safety](https://docs.agoric.com/glossary/#offer-safety) guarantees they will either receive the asking price and transfer the asset to the buyer, or maintain the right to cancel the offer (and receive their asset back). 
+
+KREAd does not impose a time limit to market entries, as long as its creator does not cancel it it will be available for a counter party to fulfill the offer.
```

Comment:
```suggestion
KREAd does not impose a time limit to market entries. As long as its creator does not cancel it, it will be available for a counter party to fulfill the offer.
```